.idea/codeStyleSettings.xml

```diff
@@ -3,10 +3,28 @@
   <component name="ProjectCodeStyleSettingsManager">
     <option name="PER_PROJECT_SETTINGS">
       <value>
-        <option name="CLASS_COUNT_TO_USE_IMPORT_ON_DEMAND" value="99" />
-        <option name="NAMES_COUNT_TO_USE_IMPORT_ON_DEMAND" value="99" />
+        <option name="OTHER_INDENT_OPTIONS">
```

Comment:
Where is the official one? Some things like the new line before package wasn't working.